LiteFarmOrg/LiteFarm
# Lf 2086 - when creating crop plan, warn user of organic status mismatch between varietal and location

To Test: 

When creating a crop plan warn of mismatches using the following 3 cases:


Case 1: Organic varietal on an organic location → No pop-up
Case 2: Non-organic varietal on a non-organic location → No pop-up
Case 3: Organic varietal on a non-organic location → Pop-up
Case 4: Non-organic varietal on an organic location → Pop-up



Case 3:
Title: “You’ve selected a non-organic location!”
Body: “You’ve indicated you’ll be planting an organic crop in a non-organic location. Do you want to proceed?”
Button: “Go back”

Clicking this button returns the user to the “Select your location” page of the crop plan and unselects the non-organic selection

Button: “That’s fine”
Clicking this button returns the user to the “Select your location” page and maintains the user’s selection

Case 4:
Title: “You’ve selected an organic location!”
Body: “You’ve indicated you’ll be planting a non-organic crop in an organic location. Do you want to proceed?”
Button: “Go back”
Clicking this button returns the user to the “Select your location” page of the crop plan and unselects the organic selection
Button: “That’s fine”
Clicking this button returns the user to the “Select your location” page and maintains the user’s selection


Pertinent edge cases:
For crop plans that will have a transplant task, the mismatch check should be made for each location
Locations that are currently “Transitioning” status should be treated as “Non-organic”

Additional context: 
This pop-up should display ANY time a mismatch occurs.

For more details: 
https://lite-farm.atlassian.net/browse/LF-2086
